uber/models.py

```diff
@@ -1,4 +1,5 @@
 from uber.common import *
+from sideboard.lib import stopped, on_startup
```

Comment:
By convention, we put all of our imports in ``common.py`` - unless the decision is made to stop doing that, this import should be moved there.